### ISSUE TITLE
Critical bug: commit message body is ignored

### ISSUE THREAD

**Issue by nerdgore** [iamchathu/changeset-conventional-commits#22]
There seems to be a bug in the code that is not picking up commits with multiple lines.
When I run the git command used to describe the commit, I do not get an error, but the body is ignored as implied by [the formatting function (`%s` only containing the subject)](https://github.com/iamchathu/changeset-conventional-commits/blob/6dd9f3a40ef75ebdce433697401121ed87eaf7ef/src/index.ts#L31).

## Impact

Crucial information may be missing from changelogs, including but not limited to message body and footers (possibly containing ticket refs and information about breaking changes).

### Current behavior

Given a commit message with a body matching the convetional commit spec
WHEN I run changeset-conventional
THEN an error is logged: `fatal: No tags can describe 'sha1'.`
THEN the change **is not added** as changeset

### Expected behavior

Given a commit message with a body matching the convetional commit spec
WHEN I run `changeset-conventional`
THEN the change **is added** as changeset

## How to reproduce

Add a commit message as allowed by the [conventional commit spec](https://www.conventionalcommits.org/en/v1.0.0/#specification).

Add a commit to your repo with the following content:
```
feat(scope): a commit subject

Add a body paragraph to explain the change.

Follow by a second paragraph as allowed by the conventional
commit spec.

Refs: #123
```

## Possible fix
Replace `%s` token in the git pretty format with `%B` to [retrieve the unwrapped message body](https://git-scm.com/docs/pretty-formats#Documentation/pretty-formats.txt-emBem) instead of the message subject.

**Comment by iamchathu** [comment]
Will you be able to work on a PR for this. Otherwise I can have a look in weekend.

**Comment by nerdgore** [comment]
I'll try my hand at it.
You don't mention contributions anywhere.
I assume add the change and cover it with tests is enough?

**Comment by iamchathu** [comment]
@nerdgore Thank you very much for fixing this. The fix is live in [v0.2.4](https://github.com/iamchathu/changeset-conventional-commits/releases/tag/v0.2.4)
